src/Umbraco.Cms.ManagementApi/Controllers/StaticFile/Tree/StaticFileTreeControllerBase.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using NSwag.Annotations;
+using Umbraco.Cms.Core;
+using Umbraco.Cms.Core.IO;
+using Umbraco.Cms.ManagementApi.Controllers.Tree;
+using Umbraco.New.Cms.Web.Common.Routing;
+
+namespace Umbraco.Cms.ManagementApi.Controllers.StaticFile.Tree;
+
+[ApiVersion("1.0")]
+[ApiController]
+[VersionedApiBackOfficeRoute("static-file/tree")]
+[OpenApiTag("StaticFile")]
+public class StaticFileTreeControllerBase : FileSystemTreeControllerBase
+{
+    private static readonly string[] _whiteListedRootFolders = { "App_Plugins", "wwwroot" };
+
+    public StaticFileTreeControllerBase(IPhysicalFileSystem physicalFileSystem)
+        => FileSystem = physicalFileSystem;
+
+    protected override IFileSystem FileSystem { get; }
+
+    protected override string FileIcon(string path) => Constants.Icons.DefaultIcon;
+
+    protected override string ItemType(string path) => "static-file";
```

Comment:
I would like that, but all other item types originate from `Constants.UdiEntityType` - i.e. `Constants.UdiEntityType.PartialView` for the partial views tree. Static files do not have an implementation representation like templates (`ITemplate`) or partial views (`IPartialView`).... wouldn't it seem weird to let static files have an UDI type then?

Reply:
Hmm yeah I see your point, and I don't think it should be there then, since it might cause confusion. 

Alternatively, we could add a new section, but I think it might be a little overkill for one type, and might also be confusing in a "what's the difference between this and an UdiEntityType" way, so maybe we should just leave it as is

Reply:
We can always refactor things later if need be 😄